Moblin/Various/Model/ModelKick.swift

```diff
@@ -121,6 +121,22 @@ extension Model {
         return KickApi(channelId: channelId, slug: slug, accessToken: stream.kickAccessToken)
     }
 
+    func searchKickCategories(query: String, onComplete: @escaping ([KickCategory]?) -> Void) {
+        createKickApi(stream: stream)?.searchCategories(query: query, onComplete: onComplete)
+    }
+
+    func setKickStreamCategory(stream: SettingsStream, categoryId: Int) {
+        guard let kickApi = createKickApi(stream: stream) else {
+            makeNotLoggedInToKickToast()
+            return
+        }
+        kickApi.setStreamCategory(categoryId: categoryId) { ok in
```

Comment:
Just do `createKickApi(stream: stream)?.setStreamCategory()`, as we do in most places.